src/Controls/src/Core/RefreshView/RefreshView.cs

```diff
@@ -135,6 +112,20 @@ public IPlatformElementConfiguration<T, RefreshView> On<T>() where T : IConfigPl
 			return _platformConfigurationRegistry.Value.On<T>();
 		}
 
+		ICommand ICommandElement.Command => Command;
+
+		object ICommandElement.CommandParameter => CommandParameter;
+
+		protected override bool IsEnabledCore => base.IsEnabledCore && CommandElement.GetCanExecute(this);
+		
+		void ICommandElement.CanExecuteChanged(object sender, EventArgs e)
+		{
+			if((bool)GetValue(IsRefreshingProperty))
```

Comment:
Just a nit for if I find any issues otherwise we can fix later (or ignore) so we don't lose the nice green checkmark on this PR.

```suggestion
			if (IsRefreshing)
```